tests/test_potential.py

```
@@ -0,0 +1,216 @@
+
+    def test_convert_path_to_abs_posix(self):
+        self.assertEqual(
+            convert_path_to_abs_posix(path="~/test"),
+            os.path.abspath(os.path.expanduser("~/test")).replace("\\", "/"),
+        )
+        self.assertEqual(
+            convert_path_to_abs_posix(path="."),
+            os.path.abspath(".").replace("\\", "/"),
+        )
+
+    @mock.patch.dict(os.environ, {"CONDA_PREFIX": "/tmp/conda"}, clear=True)
```

Ruff (0.14.4)

Probable insecure usage of temporary file or directory: "/tmp/conda"

(S108)